mailbox/hbase/src/main/java/org/apache/james/mailbox/hbase/HBaseMessageManager.java

```diff
@@ -37,8 +37,11 @@
 
 /**
  * HBase implementation of MessageManager.
- * 
+ *
+ * @Depracted: See JAMES-2571
```

Comment:
idem

Reply:
One time is enough, else you will crash GitHub :trollface:

Reply:
Intents spotted, damned :)